### ISSUE TITLE
20200924 Optimization Result

### ISSUE THREAD

**Issue by ChoiDM** [Yonsei-TAIL/KHD2020#3]
## KHD048/2020KHD_PNS/215/68
- Weight decay : 1e-3 ~ 1e-2 실험 후 적당한 5e-2 사용
- Learning rate : Initial을 1e-3으로 잡고 epoch 40과 55에서 10으로 decay
- Optimizer : SGD (momentum = 0.9)
- Network : ResNet-34 (dropout 0.5)

### To do list
- [ ] 다양한 network (EfficientNet, ResNet18)
- [ ] Bias에 weight decay 미적용 실험 (argument에 bias_decay 추가 완료)
- [ ] Learning rate cosine annealing decay
- [ ] Augmentation을 강하게 주고 실험 (cutout, cutmix, flip-ud, translation, blur)
- [ ] Class weights 실험 (class 2가 가장 F1 score가 낮음)
